### DISCUSSION TITLE
Runtime library name

### DISCUSSION THREAD

**Discussion by ryanmrichard** [NWChemEx/ParallelZone#3] [General]
The name "Runtime" is way too general for my liking. Let's come up with a new one.

Suggestions (disclaimer no idea if these names are available):
- Unparalleled
- ParallelDynamics
- ParallelPrecision
- FullSpectrumParallel
- ParallelRun
- RealtimeParallel
- MaximalParallel
- ParallelZone (Highway to the...)

**Comment by ryanmrichard** [comment]
I'm liking ParallelZone and it seems to be available.

**Reply by jwaldrop107** [reply]
I'll second this.

**Comment by ryanmrichard** [comment]
The name is settled then. Locking the "conversation".
